### PR TITLE
Locales support for disc-armory

### PR DESCRIPTION
**Pull request by rex2630**

### CODE REVIEW

**Comment by rex2630** [comment]
I have done some fixes, but i still dont see marker, any ideas?

File: disc-armory/locales/en.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Locales['en'] = {
+    ['key'] = "Press ~INPUT_CONTEXT~ to open the Armory",
+    ['take'] = "Take",
+    ['store'] = "Store",
+    ['buy'] = "Buy",
+    ['takewea'] = "Take Weapons",
+    ['already'] = "You already have a ",
+    ['took'] = "Took Weapon ",
+    ['unable'] = "Unable to take Weapon ",
```
Comment (DiscworldZA) [inline disc-armory/locales/en.lua]:
```suggestion
    ['unabletake'] = "Unable to take Weapon ",
```

File: disc-armory/locales/en.lua [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    ['failed'] = "Failed to store Weapon ",
```
Comment (DiscworldZA) [inline disc-armory/locales/en.lua]:
```suggestion
    ['failedstore'] = "Failed to store Weapon ",
```

File: disc-armory/locales/en.lua [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    ['stored'] = "Stored Weapon ",
+    ['bought'] = "You bought a ",
+    ['need'] = "You need $",
+    ['unable1'] = "Unable to buy Weapon ",
+    ['buy1'] = "Buy Weapons",
```
Comment (DiscworldZA) [inline disc-armory/locales/en.lua]:
```suggestion
    ['buyweapons'] = "Buy Weapons",
```

File: disc-armory/locales/en.lua [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    ['buy'] = "Buy",
+    ['takewea'] = "Take Weapons",
+    ['already'] = "You already have a ",
+    ['took'] = "Took Weapon ",
+    ['unable'] = "Unable to take Weapon ",
+    ['failed'] = "Failed to store Weapon ",
+    ['stored'] = "Stored Weapon ",
+    ['bought'] = "You bought a ",
+    ['need'] = "You need $",
+    ['unable1'] = "Unable to buy Weapon ",
```
Comment (DiscworldZA) [inline disc-armory/locales/en.lua]:
```suggestion
    ['unablebuyweapon'] = "Unable to buy Weapon ",
```

File: disc-armory/locales/en.lua [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    ['already'] = "You already have a ",
+    ['took'] = "Took Weapon ",
+    ['unabletake'] = "Unable to take Weapon ",
+    ['failedstore'] = "Failed to store Weapon ",
+    ['stored'] = "Stored Weapon ",
+    ['bought'] = "You bought a ",
+    ['need'] = "You need $",
+    ['unablebuyweapon'] = "Unable to buy Weapon ",
+    ['buyweapons'] = "Buy Weapons",
+    ['store1'] = "Store Weapons",
```
Comment (DiscworldZA) [inline disc-armory/locales/en.lua]:
```suggestion
    ['storeweapon'] = "Store Weapons",
```

File: disc-armory/locales/cs.lua [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    ['already'] = "Jiz mas ",
+    ['took'] = "Vzal zbran ",
+    ['unabletake'] = "Nepodarilo se vzit zbran ",
+    ['failedstore'] = "Nepodarilo se ulozit zbran ",
+    ['stored'] = "Ulozena zbran ",
+    ['bought'] = "Zakoupil jsi si ",
+    ['need'] = "Potrebujes $",
+    ['unablebuyweapon'] = "Nepodarilo se zakoupit zbran ",
+    ['buyweapons'] = "Zakoupit zbrane",
+    ['store1'] = "Ulozit zbrane",
```
Comment (DiscworldZA) [inline disc-armory/locales/cs.lua]:
```suggestion
    ['storeweapon'] = "Ulozit zbrane",
```

**Pull request merged**